### PR TITLE
docs(protocol): review protocol session

### PR DESCRIPTION
**Pull request by idea404**
Protocol section adjustments that fix typos, rephrase content, expand some content and remove duplicate information

task: https://app.asana.com/0/1206684215872507/1206684219212568/f

### CODE REVIEW

**Comment by idea404** [comment]
requires re-indexing

File: docs/site/docs/protocol/introduction/components.md [hunk 1]
```diff
@@ -2,16 +2,20 @@
...
-- [**Relayer**](../xmessages/components/relayer.md) responsible for delivering attested cross-network messages from the Omni network to destination rollup VMs. Monitors the Omni Consensus Layer until ⅔ (>66%) of the validator set attested to the “next” block on each source chain, then proceeds to forwarding the respective `XMsg` list included in the block.
+- It consists of two internal chains, a consensus layer and an execution layer, similar to post-merge Ethereum.
+    - Similar to Ethereum, Omni nodes are separated into distinct execution and consensus environments
+        - The **execution layer** is implemented by standard Ethereum execution clients. like  **`geth`**, **`erigon`**, etc, to provide the Omni EVM.
+        - The **consensus** layer is implemented by the Omni consensus client, halo, and uses CometBFT for consensus on XMsgs and Omni EVM transactions.
```
Comment (ttarsi) [inline docs/site/docs/protocol/introduction/components.md:15]:
```suggestion
        - The **consensus** layer is implemented by the Omni consensus client, halo, and uses CometBFT for consensus on XMsgs and Omni EVM blocks.
```

File: docs/site/docs/protocol/introduction/components.md [hunk 2]
```diff
@@ -2,16 +2,20 @@
...
+- **EigenLayer smart contracts** exist on Ethereum and connect the Omni network with its re-staking participants. The Omni network is registered with EigenLayer as an Actively Validated Service (AVS) and Omni validators serve the role of “Operators” of the AVS.
```
Comment (ttarsi) [inline docs/site/docs/protocol/introduction/components.md:16]: looks like we duplicate this bullet on the net line

File: docs/site/docs/protocol/xmessages/xmsg.md [hunk 3]
```diff
@@ -64,31 +66,31 @@ The following steps provide a comprehensive overview of how an XMsg travels from
...
+    $XBlock = f(chain_A)$ where $f(x)$ is a deterministic `pure` function that takes a finalized network as input and produces `XBlock` as output.
     In practice, source blocks can be streamed and transformed using a simple translation function backed by an in-memory cache.
 
     </details>
 
-    `XMsg` are associated with an `XStream`. An `XStream`  is a logical connection between a source and destination chain. It contains many `XMsg`, each with a monotonically incrementing `XStreamOffset` (the offset is like a EOA nonce, it is incremented for each subsequent message sent from a source chain to a destination chain). `XMsg`  are therefore uniquely identified and strictly ordered by their associated `XStream` and `XStreamOffset`.
+    `XMsg` are associated with an `XStream`. An `XStream`  is a logical connection between a source and destination network. It contains many `XMsg`, each with a monotonically incrementing `XStreamOffset` (the offset is like a EOA nonce, it is incremented for each subsequent message sent from a source network to a destination network). `XMsg`  are therefore uniquely identified and strictly ordered by their associated `XStream` and `XStreamOffset`.
```
Comment (ttarsi) [inline docs/site/docs/protocol/xmessages/xmsg.md:85]: Probably prefer "`XMsg`s" over "`XMsg`" since "are" is plural

**Review by ttarsi — APPROVED** [review]

**Pull request merged**